x-pack/plugins/apm/common/locator.ts

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
This and the test file can be moved to `/public` as it'll only be used in the client.

Reply:
I moved them to `public/locator` folder.